### PR TITLE
Compressed input for sailfish salmon

### PR DESCRIPTION
**Pull request by dpryan79**
Xref: https://github.com/galaxyproject/tools-iuc/issues/1128

### CODE REVIEW

File: tools/sailfish/sailfish.xml [hunk 1]
```diff
@@ -56,10 +67,24 @@
             --index $index_path
             #if $single_or_paired.single_or_paired_opts == 'single':
                 --libType ${single_or_paired.strandedness}
-                --unmatedReads ./single.$ext
+                #if $compressed == 'GZ':
+                    --unmatedReads <(zcat ./single.$ext)
+                #else if $compressed == 'BZ2':
+                    --unmatedReads <(bzcat ./single.$ext)
```
Comment (bgruening) [inline tools/sailfish/sailfish.xml]: I guess we need to add this as dependency as well, like ` bzip2`.

Comment (dpryan79) [inline tools/sailfish/sailfish.xml]: Isn't it already there on all systems?

Comment (bgruening) [inline tools/sailfish/sailfish.xml]: I would not assume anything ;)
Moreover, Galaxy supports to build containers automatically from these requirements, and then the job is running in a busybox, where this is not installed.

Comment (dpryan79) [inline tools/sailfish/sailfish.xml]: Good point. Is gzip going to be in there or does that need to be added too?

Comment (dpryan79) [inline tools/sailfish/sailfish.xml]: Is there a Galaxy bzip2 package even? If not then these become conda only tools.

Comment (dpryan79) [inline tools/sailfish/sailfish.xml]: Ah, it's package_bzlib_1_0

Comment (bgruening) [inline tools/sailfish/sailfish.xml]: Conda only is ok. Remove the tool_dependencies.xml if you lke.

Comment (dpryan79) [inline tools/sailfish/sailfish.xml]: I'll just make them conda only and be done with it.

Comment (dpryan79) [inline tools/sailfish/sailfish.xml]: Hah, they were already conda only :)

Comment (dpryan79) [inline tools/sailfish/sailfish.xml]: Should I go ahead and make gzip a dependency too? I guess then it's guaranteed to work in the busybox.

Comment (bgruening) [inline tools/sailfish/sailfish.xml]: I had a brief look, but zlib does not have the zcat binary. I was not able to find a conda package and I don't want to bug you with this so much. https://github.com/bgruening/galaxytools/issues/550

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**